frontend/src/components/mine/Reports/MineReportTable.js

```diff
@@ -72,6 +74,35 @@ const columns = [
       </div>
     ),
   },
+  {
+    title: "Documents",
+    dataIndex: "documents",
+    render: (text, record) => (
+      <div title="Documents">
+        <ul>
+          {record.report.mine_report_submissions &&
+          record.report.mine_report_submissions.length > 0 &&
```

Comment:
I think one of these checks can be removed, 

If `mine_report_submittions` is empty, is it not included in the payload as `mine_report_submittions = []`? If so, 
If it is always returned as `[]` then there is no need to check the existence.

if it isn only returned when it contains data, then there is no need to check the length as it will always be 1 or more.

Reply:
Removed the first check.